src/test/java/study/SetTest.java

```diff
@@ -0,0 +1,49 @@
+package study;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("numbers size는 3이다.")
+    void size_test() {
+        // given - when - then
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"1", "2", "3"})
+    @DisplayName("numbers 1,2,3 값이 담겨져 있다.")
```

Comment:
size_test에 남겨둔 코멘트를 참고하여 테스트케이스가 변경될 경우 DisplayName이 변경되지 않도록 변경해보면 좋겠어요!
만약 테스트데이터의 정보를 확인하고 싶다면

```suggestion
    @ParameterizedTest(name = "numbers {0} 값이 담겨져 있다.")
    @ValueSource(strings = {"1", "2", "3"})
    @DisplayName("numbers 1,2,3 값이 담겨져 있다.")
```

ParameterizedTest의 기능을 활용할 수 있습니다!

![image](https://user-images.githubusercontent.com/40811999/167441797-547f04a7-8dfd-48d7-8cd7-bda5ecf8242d.png)

Reply:
아하 가변적으로 사용이 가능하다는 것을 알게 되었네요. 감사합니다. !